### PR TITLE
Clf local options

### PR DESCRIPTION
**Pull request by occupant**

### CODE REVIEW

File: galactus.theme [hunk 1]
```diff
@@ -201,8 +201,22 @@ function galactus_preprocess_html(&$variables) {
   }
 
   // Add a class to define the active CLF theme colour
-  $clfcolour = theme_get_setting('clf_theme_colour');
-  $variables['attributes']['class'][] = 'clf-theme-' . $clfcolour;
```
Comment (joelpittet) [inline galactus.theme:205]: I worry if someone has added custom styles to this that maybe this causes issues, could we keep it or will it break things?

**Comment by joelpittet** [comment]
@occupant The "Use Minimal CLF" doesn't work if local is chosen. Maybe we can hide and disable that option if they choose local?

**Comment by joelpittet** [comment]
Screen Shot 2021-07-02 at 17 07 50-fullpage
Screen Shot 2021-07-02 at 17 07 43-fullpage
@occupant Currently something's not quite right with local, uploading screenshots of the homepage

**Comment by joelpittet** [comment]
A big part of why they are broken is mostly because I overwrote the generic `.container` classes and now local is more specific...

File: galactus.theme (hunk 1 above)
Comment (occupant) [reply galactus.theme:205]: That's a good point. Yes, best to leave this in. I'll add it back

**Comment by occupant** [comment]
> @occupant The "Use Minimal CLF" doesn't work if local is chosen. Maybe we can hide and disable that option if they choose local?

Right, the help text does note that it's just there for legacy purposes (to prevent existing settings from being lost), but hiding it altogther is a much better option.

**Comment by occupant** [comment]
> A big part of why they are broken is mostly because I overwrote the generic `.container` classes and now local is more specific...

Right, this has been an ongoin gissue (the name is just too generic and too likely to be used by other css). I'd prefer to change the markup to reference `clf-container` and the CSS accordingly, but that would necessitate a change to the `page.html.twig`. That said, I think it's a good thing to move forward with. Thoughts on that?

**Comment by joelpittet** [comment]
Some of this I can deal with because I caused it and it's meant for expert level. So I think I'm game for merging with that class added back

**Comment by joelpittet** [comment]
@occupant I'm on board with the `.clf-container` change idea, and that it only adds it when local is chosen.

**Comment by darko-hrgovic** [comment]
+1 for changing to .clf-container

**Comment by darkodevubc** [comment]
I was going to also correct "Use an external, minimal copy of the CLS [sp] css files." but I think that's going away, non?

**Pull request merged**